client/src/components/Market/index.jsx

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+const Market = () => {
+    const islands = [
+        {
+            id: 0,
+            name: "Montoya",
+            currentPrice: 367,
+            rateOfChange: 148,
+            expectedTrend: "Large Spike",
+            hasPeak: true,
+            freshPrices: true,
+        },
+        {
+            id: 1,
+            name: "Malos",
+            currentPrice: 32,
+            rateOfChange: -15,
+            expectedTrend: "Small Spike",
+            hasPeak: false,
+            freshPrices: true,
+        },
+        {
+            id: 2,
+            name: "Isla Lola",
+            currentPrice: 186,
+            rateOfChange: 24,
+            expectedTrend: "Small Spike",
+            hasPeak: true,
+            freshPrices: true,
+        },
+        {
+            id: 3,
+            name: "Mandalore",
+            currentPrice: 86,
+            rateOfChange: -12,
+            expectedTrend: "Decrementing",
+            hasPeak: false,
+            freshPrices: true,
+        },
+        {
+            id: 4,
+            name: "Raftel",
+            currentPrice: 128,
+            rateOfChange: -44,
+            expectedTrend: "Fluctuating",
+            hasPeak: false,
+            freshPrices: true,
+        },
+        {
+            id: 5,
+            name: "Ozu",
+            currentPrice: 222,
+            rateOfChange: 148,
+            expectedTrend: "Large Spike",
+            hasPeak: true,
+            freshPrices: true,
+        },
+        {
+            id: 6,
+            name: "Hyrule",
+            currentPrice: 367,
+            rateOfChange: 148,
+            expectedTrend: "Decrementing",
+            hasPeak: false,
+            freshPrices: false,
+        },
+        {
+            id: 7,
+            name: "Casatopia",
+            currentPrice: 367,
+            rateOfChange: 148,
+            expectedTrend: "Fluctuating",
+            hasPeak: false,
+            freshPrices: false,
+        },
+    ];
+
+    const sortIslandsByPeak = () => {
+        return islands.sort((a, b) =>
+            a.currentPrice <= b.currentPrice ? 1 : -1
```

Comment:
The sort function can just compare the values of `a` vs `b` here since the sort will put whichever is the lower number first

`islands.sort((a, b) => a.currentPrice - b.currentPrice)`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
🤦‍♀️ - yep. Thank you!